src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,48 +1,67 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectedTodo: Todo | null;
+  selectTodo: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, selectTodo }) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    // eslint-disable-next-line  @typescript-eslint/no-non-null-assertion
+    getUser(selectedTodo!.userId).then((response) => {
```

Comment:
- handle possible api errors
- I believe we need to add some conditions to be sure that todo is selected , you broke typescript here it's no good